pest-management-science.csl

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>Pest Management Science</title>
+    <title-short>PMS</title-short>
+    <id>http://www.zotero.org/styles/pest-management-science</id>
+    <link href="http://www.zotero.org/styles/pest-management-science" rel="self"/>
+    <link href="http://www.zotero.org/styles/spie-proceedings" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1526-4998/homepage/ForAuthors.html#preparing" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="biology"/>
+    <issn>1526-498X</issn>
+    <eissn>1526-4998</eissn>
+    <updated>2017-08-13T11:07:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="available at">available</term>
+    </terms>
+  </locale>
+  <macro name="type-number">
+    <group delimiter=" ">
+      <choose>
+        <if type="patent" match="none">
+          <text variable="genre"/>
+        </if>
+      </choose>
+      <text variable="number"/>
+    </group>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group prefix=", " delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with=". " delimiter=", " and="text"/>
```

Comment:
in the style guide I'm seeing "ed. by" for chapters and initials as for authors. Generally the chapter format is different from what you have here & below (e.g. also not seeing the square brackets around book title, different order of editor/book title). Is that on purpose based on a published piece or an oversight?